docs/essentials/collections/selectingaCollectionClass.md

```diff
@@ -1 +1,69 @@
+    
+    * One key and multiple values: Use the [System.Collections.Specialized.NameValueCollection](http://dotnet.github.io/api/System.Collections.Specialized.NameValueCollection.html) class.
+    
+* Do you need to sort the elements differently from how they were entered? 
+
+    * The [System.Collections.Hashtable](http://dotnet.github.io/api/System.Collections.Hashtable.html) class sorts its elements by their hash codes.
+    
+    * The [System.Collections.SortedList](http://dotnet.github.io/api/System.Collections.SortedList.html) class and the [System.Collections.Generic.SortedDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedDictionary%602.html) and [System.Collections.Generic.SortedList&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedList%602.html) generic classes sort their elements by the key, based on implementations of the [System.Collections.IComparer](http://dotnet.github.io/api/System.Collections.IComparer.html) interface and the [System.Collections.Generic.IComparer&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.IComparer%601.html) generic interface.
+    
+    * [System.Collections.ArrayList](http://dotnet.github.io/api/System.Collections.ArrayList.html) provides a `Sort` method that takes an `IComparer` implementation as a parameter. Its generic counterpart, the [System.Collections.Generic.List&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Generic.List%601.html) generic class, provides a `Sort` method that takes an implementation of the `IComparer<T>` generic interface as a parameter.
+    
+* Do you need fast searches and retrieval of information? 
+
+    * [System.Collections.Specialized.ListDictionary](http://dotnet.github.io/api/System.Collections.Specialized.ListDictionary.html) is faster than [System.Collections.Hashtable](http://dotnet.github.io/api/System.Collections.Hashtable.html) for small collections (10 items or fewer). The [System.Collections.Generic.Dictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.Dictionary%602.html) generic class provides faster lookup than the [System.Collections.Generic.SortedDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Generic.SortedDictionary%602.html) generic class. The multi-threaded implementation is [System.Collections.Concurrent.ConcurrentDictionary&lt;TKey, TValue&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentDictionary%602.html). [System.Collections.Concurrent.ConcurrentBag&lt;T&gt;](http://dotnet.github.io/api/System.Collections.Concurrent.ConcurrentBag%601.html) provides fast multi-threaded insertion for unordered data. For more information about both multi-threaded types, see [When to Use a Thread-Safe Collection](essentials\collections\threadsafe\whentoUseaThread-SafeCollection.md).
+    
+* Do you need collections that accept only strings? 
```

Comment:
This paragraph can just go away, since `List<string>` would now be the best way to handle this.